staging/cse/windows/azurecnifunc.ps1

```diff
@@ -458,6 +458,14 @@ function New-ExternalHnsNetwork
         $netIP = Get-NetIPAddress -ifIndex $na.ifIndex -AddressFamily IPv4 -ErrorAction SilentlyContinue -ErrorVariable netIPErr
         if ($netIP)
         {
+            $prefixOrigin = $netIP.PrefixOrigin
+            # Some VMs have multiple physical NICs where one NIC is Inteliband and should be ignored for kubelet. It's internal and used for
+            # comms with other VMs in the same cluster. Standard_HC44rs is a sample SKU. Note that this code path is not unit tested.
+            if ($prefixOrigin -ne "WellKnown" -and $prefixOrigin -ne "Dhcp") {

```

Comment:
The powershell docs here: https://learn.microsoft.com/en-us/powershell/module/nettcpip/get-netipaddress?view=windowsserver2025-ps

Mention 'Dhcp' and 'DHCP' - It looks like 'Dhcp' is the correct one otherwise if we're not 100% sure we could use 'Like' for condition?